packages/manager/modules/octavia-load-balancer/src/create/controller.js

```diff
@@ -32,6 +37,59 @@ export default class OctaviaLoadBalancerCreateCtrl {
 
   onRegionChange(region) {
     this.model.region = region;
+    this.regionSelected();
+  }
+
+  regionSelected() {
```

Comment:
**niptick** : function name is a bit confusing because it doesn't return selected region.
-> `loadPrivateNetworks()` ?